docs/execute-methods.md

```diff
@@ -1201,6 +1201,32 @@ If the connection is disconnected, condition inducer will be automatically disab
 
 Either `true` or `false`, where `true` means disabling of the condition inducer has been successful
 
+### mobile: calibrateWebToRealCoordinatesTranslation
+
+Calibrates web to real coordinates translation.
+This API can only be called from Safari web context.
+It must load a custom page to the browser, and then restore
+the original one, so don't call it if you can potentially
+lose the current web app state.
+The outcome of this API is then used in `nativeWebTap` mode.
```

Comment:
It looks like... currently `"nativeWebTapStrict": true` is also necessary to go through this logic via `click`